[with-development-client] Add EAS support (#248)
expo/examples: with-development-client/README.md

- Run `expo start --dev-client`, try it out.

## 🏗 Build with EAS

You can easily use this project with `EAS` - just follow the steps below.

### ⚙️ Prepare project

- adjust value of `ios.bundleIdentifier` and `android.package` in `app.json`
- run `eas build:configure`

### 💪 Build whatever you want

> This example comes with two pre-configured build types: `release` (a productive version of your app - ready to be uploaded to stores), `with-development-client` (a development version of your app that can be shared with your teammates).

Reviewer: ```suggestion
This example comes with two pre-configured build types: `release` (a production version of your app - ready to be uploaded to stores), `with-development-client` (a development version of your app that can be shared with your teammates).
```

> To build app, just run `eas build` and select the wanted build type.

Reviewer: ```suggestion
To build the app with the development client, just run `eas build --profile with-development-client`.
```
